db/migrate/20160407121458_remove_redundant_policy.rb

```diff
@@ -0,0 +1,15 @@
+class RemoveRedundantPolicy < ActiveRecord::Migration
+  # Remove /government/policies/schools-and-college-qualifications-and-curriculum
+  def up
+    content_items = ContentItem.where(content_id: "3c04de88-9e4a-4ebb-bdc9-ef5946db17b9")
```

Comment:
Migrations are automatically wrapped in a transaction, but you're right about the content items.  I've updated the code.